### ISSUE TITLE
[python/python3alt] JavaScript target throws "SyntaxError: Unexpected strict mode reserved word."

### ISSUE THREAD

**Issue by RossPatterson** [antlr/grammars-v4#2858]
I found this during testing of PR #2855, but it's an existing problem.  That change merely upgraded it from a hidden error to a test case failure.  For example, the checks for PR #2854 [show the error](https://github.com/antlr/grammars-v4/actions/runs/3171329768/jobs/5164665386#step:17:6461).  My change compares the error with the expected errors in [`python/python3alt/examples/shaldq.py.errors`](https://github.com/antlr/grammars-v4/blob/master/python/python3alt/examples/shaldq.py.errors) and finds that they are unexpectedly different.

Here's the test output from PR #2854:
```
Test case: D:\a\grammars-v4\grammars-v4\python\python3alt\examples\shaldq.py
--- Testing file D:\a\grammars-v4\grammars-v4\python\python3alt\examples\shaldq.py ---
file:///D:/a/grammars-v4/grammars-v4/python/python3alt/Generated/AltPython3Parser.js:481
          static protected final int ALREADY_THROWN_TOKEN_TYPE = Integer.MIN_VALUE + 23;
          ^^^^^^

SyntaxError: Unexpected strict mode reserved word
    at ESMLoader.moduleStrategy (node:internal/modules/esm/translators:139:18)
    at ESMLoader.moduleProvider (node:internal/modules/esm/loader:236:14)
Test completed, time: 00:00:00.2555092
```

**Comment by RossPatterson** [comment]
This also fails in [build-php](https://github.com/antlr/grammars-v4/actions/runs/3171329768/jobs/5164663975#step:8:6086).  After looking at the grammar, it's obvious - it contains Java code in both `@parser::members { ... }` and `@lexer::members { ... }`.  I'll add it to the skip lists for all the other targets.

This grammar is a good example of why I wanted to add code to check the error messages.  It's been "passing" for some time, even though it won't compile.

**Comment by RossPatterson** [comment]
Fixed by commit e9ed25fc737fafecfd6f822e212bbc2349bc02de.
